app/models/ems_folder.rb

```diff
@@ -25,6 +24,8 @@ class EmsFolder < ApplicationRecord
   # TODO: Vmware specific - Fix when we subclass EmsFolder
 
   def provider_object(connection)
+    # TODO once EmsFolders are subclassed this can be moved to the vmware repo
```

Comment:
Fixed in https://github.com/ManageIQ/manageiq/pull/19534/files#diff-4024dcf71764353249c6a46da6726effL22-L34